main.js

```diff
@@ -0,0 +1,29 @@
+const hamburger = document.querySelector('.hamburger');
+
+const menuLinks = document.querySelector('.menu__links');
+
+// adding the class active when clicking to the hambuger and menuLinks elements
+
+hamburger.addEventListener('click', () => {
+  document.querySelectorAll('.bar').forEach((bar) => {
+    if (bar.style.backgroundColor === 'white') {
+      bar.style.backgroundColor = '#6070ff';
+    } else {
+      bar.style.backgroundColor = 'white';
+    }
+  });
```

Comment:
- [ ] I have noticed that after clicking a link he `hamburger` button becomes invisible even though it is still there. I have recorded a short video to demonstrate that

https://user-images.githubusercontent.com/77038610/150943409-964b71db-75a7-4eb0-ac2d-a454db4ff5fc.mp4

- please make the `hamburger` button visible 😉